snuba/cli/jobs.py

```diff
@@ -0,0 +1,21 @@
+from typing import Tuple
+
+import click
+
+from snuba.manual_jobs.ToyJob import ToyJob
+
+
+@click.command()
+@click.option(
+    "--dry_run",
+    is_flag=True,
+    default=True,
+)
+@click.argument("pairs", nargs=-1)
+def jobs(*, dry_run: bool, pairs: Tuple[str, ...]) -> None:
```

Comment:
I think what you're doing here is a subcommand like `snuba jobs run`

I could imagine, for example, there also being e.g. a `snuba jobs list` or `snuba jobs status`